scheming/index.js

```diff
@@ -0,0 +1,154 @@
+const { BoxPlot, computeStats } = require("@vx/boxplot");
+const { scaleLinear } = require("@vx/scale");
+const { Group } = require("@vx/group");
+const ReactDOMServer = require("react-dom/server");
+const React = require("react");
+const parseUrl = require("parse-url");
+
+addEventListener("fetch", event => {
+  event.respondWith(handleRequest(event.request));
+});
+
+/**
+ * Fetch and log a request
+ * @param {Request} request
+ */
+async function handleRequest(request) {
+  let response;
+  if (request.method === "GET") {
+    const stringJson = parseUrl(request.url).query.json;
+    if (stringJson === undefined) {
+      return new Response("Expected JSON data in url for GET request", {
+        status: 400
+      });
+    }
+    const json = JSON.parse(stringJson);
+    if (json.chartType !== "boxplot") {
+      return new Response(`Unsupported chartType: ${json.chartType}`, {
+        status: 500
+      });
+    }
+    if (json.data == undefined) {
+      return new Response(`Expected JSON field: data`, {
+        status: 500
+      });
+    }
+
+    const output = ReactDOMServer.renderToString(
```

Comment:
Two things:

1. `renderToString` is intended for producing markup that will eventually be hydrated via a client-side React. Instead, `renderToStaticMarkup` is for producing plain HTML/SVG that will be used completely outside of React.

```suggestion
    const output = ReactDOMServer.renderToStaticMarkup(
```

2. React also supports streaming server-side rendering, which you can pipe to the response. This way, the server can begin sending the response incrementally during render, without first having to wait for the entire render to complete.

I think Cloudflare Workers supports streaming responses: https://workers.cloudflare.com/docs/reference/runtime/apis/streams/

In this case you would use `React.renderToStaticNodeStream`: https://reactjs.org/docs/react-dom-server.html#rendertostaticnodestream